routes/checkout.js

```diff
@@ -2,7 +2,12 @@ const express = require('express');
 const router = express.Router();
 
 router.get('/', function(req, res, next) {
-    res.render('checkout', {title: "OnlyDucks Checkout"})
+    if(req.session.authenticatedUser) {
+        res.render('checkout', {title: "OnlyDucks Checkout"})
+    } else{
+        req.session.loginMessage = "Please login to complete your order."
```

Comment:
![Screen Shot 2022-12-05 at 9 25 37 PM](https://user-images.githubusercontent.com/77898527/205823817-e1553409-b690-4901-8aa4-33687cf210d6.png)
I feel like this is slightly doubled with the text, but ik that's super nitpicky, more of an observation